core/lib/libtomcrypt/sub.mk

```diff
@@ -114,6 +114,7 @@ _CFG_CRYPTO_WITH_ASN1 := $(call cryp-one-enabled, RSA DSA ECC)
 _CFG_CRYPTO_WITH_FORTUNA_PRNG := $(call cryp-all-enabled, AES SHA256)
 
 cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -DLTC_SMALL_CODE -DLTC_NO_FAST
+cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -Os
```

Comment:
My mistake :angry:  Sorry for the noise.

About `LTC_NO_FAST`, it has no impact on ARM. Here is a comment from `tomcrypt_cfg.h`:

    Note: in order to use the optimized macros your platform must support
    unaligned 32 and 64 bit read/writes. The x86 platforms allow this but some
    others [ARM for instance] do not.  On those platforms you **MUST** use
    the portable [slower] macros.

Looking at some code, it looks like some optimizations can be done also for ARM. But as I am missing simple `-Os` compilation option today, I won't do that now for sure!

Conclusion: I'll leave 1st 2 commits, remove the 3rd one with the `-Os`, and remove `LTC_NO_FAST` which is not relevant.